ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB.hs

```diff
@@ -9,10 +11,10 @@ import qualified Test.Ouroboros.Storage.ChainDB.Paths as Paths
 import qualified Test.Ouroboros.Storage.ChainDB.StateMachine as StateMachine
 
 tests :: TestTree
-tests = testGroup "ChainDB" [
+tests = testGroup "ChainDB" $ [
       Iterator.tests
     , GcSchedule.tests
     , Model.tests
     , Paths.tests
-    , StateMachine.tests
-    ]
+    ] <>
```

Comment:
Would it make sense to comment on why this test was disabled? Maybe transfer the PR template comment here. Also we might want to link to the issue that refers to fixing and enabling these tests.

Reply:
Right, will add a comment, that is nicer than having to go through `git blame` :+1: